backend/src/ee/services/access-approval-request/access-approval-request-dal.ts

```diff
@@ -141,7 +143,7 @@ export const accessApprovalRequestDALFactory = (db: TDbClient) => {
               }
             : null,
 
-          isApproved: !!doc.privilegeId
+          isApproved: !!doc.policyDeletedAt || !!doc.privilegeId
```

Comment:
This will resolve to `true` if the policy is not deleted. We are basically saying: isApproved = true IF policy not deleted, OR privilege ID present

Reply:
`!!doc.policyDeletedAt` will resolve to false if the value is null and true if the value is set
![CleanShot 2024-12-09 at 12 44 31@2x](https://github.com/user-attachments/assets/d5376e1b-61a1-4474-8eae-80be8efd3ae0)

We use this value to show the difference on the UI and block changes from happening
if we do not return as "isApproved" it will show in the UI that its an outstanding request and offer the user to approve etc.

We can make separate front end changes to avoid this if we need